persistence/jpa/src/main/java/org/infinispan/persistence/jpa/configuration/JpaStoreConfigurationSerializer.java

```diff
@@ -0,0 +1,32 @@
+package org.infinispan.persistence.jpa.configuration;
+
+import javax.xml.stream.XMLStreamException;
+
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+import org.infinispan.configuration.serializing.AbstractStoreSerializer;
+import org.infinispan.configuration.serializing.ConfigurationSerializer;
+import org.infinispan.configuration.serializing.XMLExtendedStreamWriter;
+
+/**
+ * JpaStoreConfigurationSerializer.
+ *
+ * @since 9.0
+ * @author Tristan Tarrant
+ */
+public class JpaStoreConfigurationSerializer extends AbstractStoreSerializer implements ConfigurationSerializer<JpaStoreConfiguration> {
+
+   @Override
+   public void serialize(XMLExtendedStreamWriter writer, JpaStoreConfiguration configuration) throws XMLStreamException {
+      AttributeSet attributes = configuration.attributes();
+      writer.writeStartElement(Element.JPA_STORE);
+      attributes.write(writer, JpaStoreConfiguration.ENTITY_CLASS, Attribute.ENTITY_CLASS_NAME);
+      attributes.write(writer, JpaStoreConfiguration.PERSISTENCE_UNIT_NAME, Attribute.PERSISTENCE_UNIT_NAME);
+      attributes.write(writer, JpaStoreConfiguration.BATCH_SIZE, Attribute.BATCH_SIZE);
+      attributes.write(writer, JpaStoreConfiguration.STORE_METADATA, Attribute.STORE_METADATA);
+
+      writeCommonStoreSubAttributes(writer, configuration);
+      writeCommonStoreElements(writer, configuration);
```

Comment:
this can be removed.
the JPA parser does not parse write-behind neither property (throws parse exception) and IntelliJ gives an error saying that elements does not belong there.
Although, I'm not sure if the problem is here or the parser :)